src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -312,7 +312,8 @@ inline static mkldnn::memory::desc GetFCWeightDesc(const NDArray &arr, int dtype
   for (size_t i = 0; i < dims.size(); i++) dims[i] = arr.shape()[i];
   auto format = mkldnn::memory::format_tag::any;
   // for batch 256 alexnet benchmark test
-  if (dims.size() == 2) {
+  const bool brgemm_disabled = dmlc::GetEnv("MXNET_DISABLE_ONEDNN_BRGEMM_FC", true);
```

Comment:
It will be good to add description to docs/static_site/src/pages/api/faq/env_var.md
Also I am not sure if for 1.x branch the name have to include ONEDNN ?
so maybe MXNET_MKLDNN_DISABLE_BRGEMM_FC